main/xf86-video-qxl/APKBUILD

```diff
@@ -36,6 +38,18 @@ build() {
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info \
 		--localstatedir=/var \
+		--enable-xspice=only \
+		--disable-kms \
+		|| return 1
+	make || return 1
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		--localstatedir=/var \
 		|| return 1
 	make || return 1
```

Comment:
This is almost certainly wrong. The second run of make just overwrites all the files generated by the previous run.